charts/traffic-proxy/templates/configmap.yaml

```
@@ -44,12 +44,21 @@ data:
                                     '"request_id":"$http_x_request_id",'
                                     '"upstream_request_id":"$upstream_http_x_request_id"}';
 
+      {{- if .Values.proxy.log.customFormats }}
+        {{ .Values.proxy.log.customFormats | indent 8 }}
+      {{- end }}
+
+      {{- if .Values.proxy.log.accessLog.enabled }}
+        access_log {{ .Values.proxy.log.accessLog.path }}  {{ .Values.proxy.log.accessLog.format }};
```

```suggestion
        access_log {{ .Values.proxy.log.accessLog.path }} {{ .Values.proxy.log.accessLog.format }};
```

---

Еще вопрос, зачем нам управлять `.Values.proxy.log.accessLog.path`?
```suggestion
        access_log {{ .Values.proxy.log.accessLog }};
```

в values добавить дефолт `accessLog: off` И тогда не нужны эти if

---

чтобы управлять в том числе форматом логов

---

Если переделать в строку,
accessLog: "/dev/stdout main"
то можно в том числе управлять и форматом, но не будет if в шаблоне

---

изменил
errorlog так же изменил, шансы что путь до файла лога будет меняться крайне малы